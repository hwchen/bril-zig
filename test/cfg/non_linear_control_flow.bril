../../bril/test/interp/core/non_linear_control_flow.bril