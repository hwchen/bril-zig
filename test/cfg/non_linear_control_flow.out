{"0":["start"],"end":[],"l_1_3":["end"],"l_1_2":["l_1_3"],"l_1_1":["l_1_3"],"l_0_3":["l_1_1","l_1_2"],"l_0_2":["l_0_3"],"l_0_1":["l_0_3"],"start":["l_0_1","l_0_2"]}
